src/OpenTK.Mathematics/Geometry/Box2.cs

```diff
@@ -261,28 +261,29 @@ public Box2 Scaled(Vector2 scale, Vector2 anchor)
         }
 
         /// <summary>
-        /// Inflate this Box2 to encapsulate a given point.
+        /// Inflates this Box2 by the given size in all directions. A negative size will shrink the box to a maximum of -HalfSize.
+        /// Use the <see cref="Extend"/> method for the point-encapsulation functionality in earlier API versions.
```

Comment:
I think "earlier API versions" should be more specific about versions.
Something like `pre 4.8.0` or whichever function deprecated the old functionality.

Reply:
Updated all references to `OpenTK version 4.8.1 and earlier`